src/plugins/autoplay/autoplay.js

```diff
@@ -49,6 +49,16 @@
         // triggered for the first slide, so that's where code flow continues.
     }, false );
 
+    document.addEventListener( "impress:autoplay:pause", function( event ) {
+        status = "paused";
+        reloadTimeout( event );
+    }, false );
+
+    document.addEventListener( "impress:autoplay:resume", function( event ) {
```

Comment:
Could you change this event to "impress:autoplay:play".

Reply:
done =D